test-three/test-three/src/Content/Content.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import{Route, NavLink, HashRouter} from "react-router-dom";
+import Pages from'./Pages';
+import Links from'./Links';
+import Widgets from'./Widgets';
+
+import PostData from '../data/menu-data.json';
+
+class Content extends Component {
```

Comment:
Is there a way this solution could be built without duplicating the menu code for each section of the website?

Reply:
Yes, thinking about it again I could of just made a menu component that would load the subheadings and then change the url from there